registration/api/v2/serializers.py

```diff
@@ -180,8 +180,10 @@ def __init__(self, *args, **kwargs):
         """
         super().__init__(*args, **kwargs)
 
+        # Just using the new v1 schema for now, adding the v0 schema breaks
+        # code generation because one variable is named main_color and other is
+        # main-color and both get converted to mainColor in TS
```

Comment:
if we must support v0 and v1 and this is a temp solution, shouldn't you add a TODO here?

Reply:
We actually plan to remove all v0 related code after this is deployed to production, so it doesn't make much sense to implement it.